src/pretix/base/models/items.py

```diff
@@ -111,6 +110,33 @@ class ItemCategory(LoggedModel):
                     'only be bought in combination with a product that has this category configured as a possible '
                     'source for add-ons.')
     )
+    CROSS_SELLING_MODES = (
+        (None, _('Normal category')),
+        ('both', _('Combined category')),
+        ('only', _('Cross-selling category')),
+    )
+    cross_selling_mode = models.CharField(
+        choices=CROSS_SELLING_MODES,
+        null=True,
+        max_length=5
+    )
+    CROSS_SELLING_CONDITION = (
+        ('always', _('Always show in cross-selling step')),
+        ('products', _('Only if the cart contains one of these products')),
+        ('discounts', _('Only show products affected by discount rules')),
```

Comment:
I'm not sure if "affected" is totally clear here to the user, but I have not yet come up with a better idea.

Reply:
In line with your "celebration" wording from Slack maybe "Only show products that qualify for a discount according to discount rules"?

Reply:
:+1: